Open_Decision_Framework-community-v1.0.md

```diff
@@ -0,0 +1,328 @@
+  - Who do we need more buy-in or support from?
+
+**Key considerations**
+
+  - Representation of different types of customers
+  - Unexpected impacts and use cases
+  - Unspoken risks and concerns
+
+Phase: Launch
+-------------
+
+**Steps you can take to be open**
+
+*Begin with the end in mind*
+
+  - Demonstrate alignment with Red Hat's strategy, mission, culture, and values
+  - Outline the steps you've taken to make this decision openly
+  - Highlight use of this framework
+  - Tell associates where to find detailed information
+  - Show how feedback shaped the decision or project
+  - Explain how to provide input after launch
+  - Acknowledge when you're not fully satisfied with the decision or know that others will not be
+  - Share your timeline or criteria for revisiting the decision
+  - Stay engaged with those who reject the decision
+
+*Default to open*
+
+  - Reiterate relevant business requirements and constraints
+  - Share relevant legal, reporting, or confidentiality issues
+  - Communicate success criteria and publish relevant metrics
+
+*Contribute upstream*
+
+  - Publish your methods, lessons learned, communications, and decision criteria to the archive, so others can review past decisions, learn why a decision was made, and see how leaders have responded to similar issues in the past
+  - Offer guidance to others on open decision making and choosing collaboration tools
+
+**Questions to ask**
+
+  - How will we monitor mailing lists and other feedback channels after the launch?
+  - If we have done early releases, will we continue to make incremental improvements based on feedback?
+  - How willing are we to make revisions based on feedback?
+  - What's a reasonable window of time for additional input and refinement?
+  - Did we overlook something important? How do we address it?
+  - Does the decision need to be revisited?
+  - Did open decision-making lead to the desired outcomes?
+  - How can we share our lessons learned and encourage open decision-making at Red Hat?
+
+RESOURCES 
+=========
+
+  - [The Open Source Way handbook](https://www.theopensourceway.org/wiki/Introduction) –  guide to creating and nurturing communities of contributors
+  - [The Open Organization](http://opensource.com/open-organization) (book + online community)
+  - [Prioritizing by impact](http://developerblog.redhat.com/2014/08/27/5-ux-tips-for-developers/), see grid in Máirín Duffy's "5 UX Tips for Developers"
+  - [Opensource.com](http://www.opensource.com/) – A Red Hat supported publication focused on how open source principles can be applied to business, education, government, and more
+  - [The Advice Process](http://danieltenner.com/2014/11/06/the-advice-process-definition-and-usage-tips/) (Daniel Tenner)
+
+APPENDIX 
+========
+
+History: Where the Open Decision Framework came from
+----------------------------------------------------
+
+  - Based on principles practiced by open source communities
+
```

Comment:
@semioticrobotic They were nested in the ODP, so I tried to preserve that formatting.